Tile for Email Routing (#2626)
cloudflare/cloudflare-docs: products/email-routing/src/content/index.md

---
title: Overview
order: 0
pcx-content-type: landing-page
---

# Cloudflare Email Routing

Cloudflare Email Routing is designed to simplify the way you create and manage email addresses, without needing to keep an eye on additional mailboxes. With Email Routing, you can create any number of custom email addresses that you can use in situations where you do not want to share your primary email address, such as when you subscribe to a new service or newsletter. Emails are then routed to your preferred email inbox, without you ever having to expose your primary inbox.

> Email Routing is free, private by design and Cloudflare doesn’t store or access the emails routed to your inbox.

Reviewer: ```suggestion
Email Routing is free for all Cloudflare users and private by design. Cloudflare will not store or access the emails routed to your inbox.
```
